### ISSUE TITLE
project update #7

### ISSUE THREAD

**Issue by loreese** [loreese/sherlock#15]
This week our group wasn't able to assemble as a team, but we did get a chance to meet over spring break to discuss our midterm work plans. We really haven't discussed much beyond our midterm work besides combing back through our text mark-up to make sure our collective mark-up decisions have a harmony. Moving forward, I think we are just trying to get through this weekend's workload for our hard term goals.

**Comment by Ethandaniel47** [comment]
I think everyone is just trying their best right now to get through the hump of midterm goals. I applaud your group for taking the initiative to meet over spring break for something of a status update. I think that the break threw a bit of a wrench in our progress between travel and trying to take a genuine break we got a little behind and have kicked it into high gear this weekend to resolve our issues and get everything done by Monday.

**Comment by emmamamula** [comment]
I agree, it is admirable that you met over break and it I'm sure it helped to maintain the consistency of work flow. Our group has also focused on completing our midterm goals and it seems that for your group most of your markup and tedious project work will be complete soon.

**Comment by rmf96** [comment]
What kinds of things did you do to make sure that your collective mark-up decisions had harmony besides validating against a schema? Our group has been having a little trouble with this

**Comment by k-busko** [comment]
We have also had some deliberation within our group about maintaining this harmony (good word by the way) for xml markup. We sometimes had to sit down and talk in person about the decisions we were making about what parts of our texts received the designated tagging. Its a great idea that you guys are going back through your mark up and revising/ touching up on anything you may have missed.

**Comment by BenBavar** [comment]
@rmf96 @k-busko What kinds of harmony, in addition to validating against a common schema, do you two consider to be pertinent to the mark-up for our projects? I am not sure what you have in mind.
